src/index.ts

```diff
@@ -2,32 +2,34 @@
 
```

Comment:
No logical changes here. Mostly var -> const/let, and adding some types where I could figure them out quickly